src/utils.rs

```diff
@@ -113,6 +113,18 @@ where
     })
 }
 
+pub fn de_location_trim_with_default<'de, D>(deserializer: D) -> Result<f64, D::Error>
+    where
+        D: ::serde::Deserializer<'de>,
+{
+    use serde::Deserialize;
+    let s = String::deserialize(deserializer)?;
+    Ok(s.trim().parse::<f64>().unwrap_or_else(|e| {
+        error!("{}", e);
+        0.00
```

Comment:
Not a fan of putting stop_point on the null island, but if it's ok for @prhod I'am ok too

Reply:
not a fan too but he made me do it :p

Reply:
@kinnou02  I'm open to any other solution that enable conversion (without panick)  and if possible not much refacto ^^ 
Using a blank value doesn't seem to be a good solution :/